.github/PULL_REQUEST_TEMPLATE/PRNewFeature_Template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+Please describe the new feature. 
+Include a summary of the change(s) you're making.
+
+If this fixes an issue:
+Fixes # (issue) 
+
+## Type of change
+- [ ] New feature (without breaking existing functionality)
+
+# How Has This Been Tested?
+
+If you ran any test, please describe the tests that you ran to verify your changes. 
+Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] Finish my changes
+- [ ] My changes generate no new errors or warnings
+- [ ] Push my commits to GitHub
+- [ ] Open a pull request
```

Comment:
Maybe this could be erased? commits and pull request, I mean this is the pull request template

Reply:
Yes, you're right.